modules/png-extraction/ImageExtractor.py

```diff
@@ -46,6 +46,8 @@ def initialize_config_and_execute(config_values):
     send_email = bool(configs['SendEmail'])
     no_splits = int(configs['SplitIntoChunks'])
     is16Bit = bool(configs['is16Bit']) 
+    global publicheadersonly
```

Comment:
Can you please define this global with the rest of the global contents above, rather than here?

Reply:
Please recreate your pull request against the dev branch. Pull requests must always be against the dev branch.

Please read https://github.com/Emory-HITI/Niffler/blob/dev/CONTRIBUTING.md